Whole file `# type: ignore` does not silence import errors

I just discovered PR #6830 and tried it out in master. It seems to work for the most part, but Mypy still complains about missing stub files. For example:

```
# type: ignore
import boto3 # This does not get ignored.
n:List[int] = 'a' # This gets ignored.
```

```
x.py:2: error: No library stub file for module 'boto3'
x.py:2: note: (Stub files are from https://github.com/python/typeshed)
```

It seems missing a critical stub file is a good use case for ignoring an entire file, but perhaps I'm missing something. Hope this helps!

version: mypy 0.710+dev.7cef06bc65d416ed3a533bbc4c630bd9417adb06.dirty

--- a/mypy/fastparse.py
+++ b/mypy/fastparse.py
@@ -38,6 +38,7 @@ from mypy import defaults
 from mypy import message_registry
 from mypy.errors import Errors
 from mypy.options import Options
+from mypy.reachability import mark_block_unreachable
 
 try:
     # pull this into a final variable to make mypyc be quiet about the
@@ -309,7 +310,7 @@ class ASTConverter:
                 and min(self.type_ignores) < self.get_lineno(stmts[0])):
             self.errors.used_ignored_lines[self.errors.file].add(min(self.type_ignores))
             block = Block(self.fix_function_overloads(self.translate_stmt_list(stmts)))
-            block.is_unreachable = True
+            mark_block_unreachable(block)
             return [block]
 
         res = []  # type: List[Statement]

--- a/mypy/fastparse2.py
+++ b/mypy/fastparse2.py
@@ -47,6 +47,7 @@ from mypy import message_registry
 from mypy.errors import Errors
 from mypy.fastparse import TypeConverter, parse_type_comment, bytes_to_human_readable_repr
 from mypy.options import Options
+from mypy.reachability import mark_block_unreachable
 
 try:
     from typed_ast import ast27
@@ -207,7 +208,7 @@ class ASTConverter:
                 and min(self.type_ignores) < self.get_lineno(stmts[0])):
             self.errors.used_ignored_lines[self.errors.file].add(min(self.type_ignores))
             block = Block(self.fix_function_overloads(self.translate_stmt_list(stmts)))
-            block.is_unreachable = True
+            mark_block_unreachable(block)
             return [block]
 
         res = []  # type: List[Statement]

--- a/test-data/unit/check-ignore.test
+++ b/test-data/unit/check-ignore.test
@@ -242,6 +242,10 @@ IGNORE
 def f(): ...
 IGNORE
 
+[case testIgnoreWholeModule5]
+# type: ignore
+import MISSING
+
 [case testDontIgnoreWholeModule1]
 if True:
     # type: ignore

